### DISCUSSION TITLE
可以在播放器上添加 视频旋转 功能吗？

### DISCUSSION THREAD

**Discussion by kethoven** [the1812/Bilibili-Evolved#2010] [功能建议 / Ideas]
可以在播放器上添加 视频旋转 功能吗？

**Comment by the1812** [comment]
这应该很少有用到吧(
要用的时候直接控制台里
```js
document.querySelector('video').style.transform = 'rotate(180deg)'
```
旋转的角度可以随便改

**Reply by kethoven** [reply]
> 这应该很少有用到吧(
> 要用的时候直接控制台里
> 
> ```js
> document.querySelector('video').style.transform = 'rotate(180deg)'
> ```
> 
> 旋转的角度可以随便改

可以增加为一个快捷键吗？如shift+R。

**Reply by the1812** [reply]
等自定义快捷键动作做了就能加

**Reply by kethoven** [reply]
> 等自定义快捷键动作做了就能加

那太好了！
